vendor/github.com/pingcap/parser/ast/dml.go

```diff
@@ -739,11 +739,13 @@ func (n *InsertStmt) Accept(v Visitor) (Node, bool) {
 		n.Select = node.(ResultSetNode)
 	}
 
-	node, ok := n.Table.Accept(v)
-	if !ok {
-		return n, false
+	if n.Table != nil {
```

Comment:
Could you please give an example where `n.Table` is nil in `INSERT`?

Reply:
I don't know a real example. The code is to pass the test case below .
https://github.com/pingcap/tidb/blob/6fbb42d22bdfdbe4822174c44430b64f7946b73f/planner/core/cacheable_checker_test.go#L41-L42

Reply:
I see, then this code change is unreasonable IMHO. By the way, I learned that since we have moved parser to a separate repository, we need to file a PR in https://github.com/pingcap/parser if we want to make changes, and then `make update` in tidb repo.